lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,10 +281,23 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        cert = open(module.params.get('cert'), 'r').read().rstrip()
```

Comment:
The changes we've made to this part of the module is responsible for the conflicts with https://github.com/ansible/ansible/commit/62fa2019c6846def66912b94ba1b5ed0c40525d and will be removed when rebasing. Should I go ahead and rebase so just changes for arn will remain?